protractor.conf.js

```diff
@@ -32,7 +32,7 @@ exports.config = {
     });
     jasmine.getEnv().addReporter(new SpecReporter({ spec: { displayStacktrace: true } }));
     browser.params.user = user.get();
-    
```

Comment:
Hmm...my IDE automatically removes spaces in empty lines creating this change.  @lmmrssa Go ahead and change this if you want.

Reply:
yes this is good we should not have spaces on empty line anyway.